### ISSUE TITLE
Custom fluids cause rendering performance drop.

### ISSUE THREAD

**Issue by tgarr0** [mraof/Minestuck#94]
Close proximity to large bodies of custom fluids (such as a brain juice ocean) will cause rendering lag while travelling with it in view.  This is also a problem in versions prior to the switch to Forge fluid rendering. I tried to find other available 1.8 mods with custom fluids, and found only DeadMess; their blood oceans cause the same slowdown.
This may very well be an issue to pass on to Forge, but it seemed like a good idea to bring it up here first.

**Comment by kirderf1** [comment]
I think I have an idea what forge is doing poorly. And if I'm correct, minestuck didn't have that problem when I implemented fluid rendering. (and before I moved to the fluid rendering used by forge)
So, I believe that the problem lies in that forge creates a new model every time it looks for one, while the system I created used a cache to avoid repeatedly creating identical models.

I'll get on to submit an issue to forge tomorrow.
